src/Objs/Homepage/SiteColorsPicker.tsx

```diff
@@ -44,6 +44,30 @@ export const SiteColorsPicker = connect(function SiteColorsPicker({
         </div>
       </div>
 
+      <div className="row">
+        <div className="col-sm-6">
+          <div className="scrivito_detail_label">
+            <span>Body text color</span>
```

Comment:
* [ ] FYI the word "color" is redundant for "Body text" and "Headline text". I'd even go with "Headlines" instead of "Headline text color".

Reply:
I went with `Body text` and `Headline text`. I kept the `text` for Headline to reduce mental capacity (which headline color is this referring to? The text color or the background color).